### ISSUE TITLE
String -> UIWindowSceneSessionRole cast error with Xcode11 updates

### ISSUE THREAD

**Issue by whitneyschmidt** [xamarin/xamarin-macios#6702]
### Steps to Reproduce

1. 
```
using UIKit;
namespace singleview
{
        public override UISceneConfiguration GetConfiguration(UIApplication application, UISceneSession connectingSceneSession, UISceneConnectionOptions options)
        {
            return new UISceneConfiguration("DefaultConfiguration", connectingSceneSession.Role);
        }

```
2. Build error:
```/Users/whitneyschmidt/Projects/iostemplate/singleview/AppDelegate.cs(69,69): Error CS1503: Argument 2: cannot convert from 'string' to 'UIKit.UIWindowSceneSessionRole' (CS1503) (singleview)```

### Expected Behavior

`UISceneSession.Role` should return a value of type `UIKit.UIWindowSceneSessionRole`, not string.

### Environment

```
=== Visual Studio Enterprise 2019 for Mac ===

Version 8.2 (build 1124)
Installation UUID: 331e7c2a-8be8-4024-9857-89c6089e765f
	GTK+ 2.24.23 (Raleigh theme)
	Xamarin.Mac 5.14.0.85 (d16-2-fix-mac-updates / 86973979)

	Package version: 600000311

=== Mono Framework MDK ===

Runtime:
	Mono 6.0.0.311 (2019-02/494641b300c) (64-bit)
	Package version: 600000311

=== NuGet ===

Version: 5.1.0.6013

=== .NET Core ===

Runtime: /usr/local/share/dotnet/dotnet
Runtime Versions:
	2.1.12
	2.1.11
	2.1.9
SDK: /usr/local/share/dotnet/sdk/2.1.701/Sdks
SDK Versions:
	2.1.701
	2.1.700
	2.1.505
MSBuild SDKs: /Library/Frameworks/Mono.framework/Versions/6.0.0/lib/mono/msbuild/Current/bin/Sdks

=== Xamarin.Profiler ===

Version: 1.6.11
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

=== Updater ===

Version: 11

=== Apple Developer Tools ===

Xcode 11.0 (14897.4)
Build 11M374r

=== Xamarin.Android ===

Not Installed

=== Microsoft Mobile OpenJDK ===

Java SDK: Not Found

Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

=== Android Device Manager ===

Version: 1.2.0.58
Hash: d03e3ea
Branch: remotes/origin/d16-2~1
Build date: 2019-07-19 20:55:38 UTC

=== Xamarin Inspector ===

Version: 1.4.3
Hash: db27525
Branch: 1.4-release
Build date: Mon, 09 Jul 2018 21:20:18 GMT
Client compatibility: 1

=== Xamarin.Mac ===

Version: 5.99.2.35 (Visual Studio Enterprise)
Hash: 527e56684
Branch: xcode11-healthkit
Build date: 2019-08-01 14:39:02-0400

=== Xamarin.iOS ===

Version: 12.99.2.35 (Visual Studio Enterprise)
Hash: 527e56684
Branch: xcode11-healthkit
Build date: 2019-08-01 14:39:02-0400

=== Xamarin Designer ===

Version: 16.2.0.391
Hash: 216490cf5
Branch: remotes/origin/d16-2-xcode11
Build date: 2019-07-25 17:42:00 UTC

=== Build Information ===

Release ID: 802001124
Git revision: 0797bb4dfcd2cbad030d7fce609e247e23f855ca
Build date: 2019-07-26 12:13:29+00
Build branch: release-8.2-xcode11
Xamarin extensions: d663a25405cd5e7d0b0deefd2a26d548680f16f5

=== Operating System ===

Mac OS X 10.14.5
Darwin 18.6.0 Darwin Kernel Version 18.6.0
    Thu Apr 25 23:16:27 PDT 2019
    root:xnu-4903.261.4~2/RELEASE_X86_64 x86_64


```

**Comment by whitneyschmidt** [comment]
@VincentDondain FYI

**Comment by VincentDondain** [comment]
Fixed

**Comment by rolfbjarne** [comment]
Closing since this was fixed in #6703.
